tools/local-test/scripts/steps/clean-outdated.sh

```
@@ -15,10 +15,10 @@ if [ -f $hash_file_name ]; then
 fi
 
 if [[ "$current_hash" != "$old_hash" ]]; then
-    print_warn "$PROJECT_NAME, исходники изменены. Будет выполнена сбора"
+    print_warn "$PROJECT_NAME, исходники ${BOLD}изменены${RESET}. Будет выполнена сборка"
     rm -rf $EXPORT_PATH/$PROJECT_NAME
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)